modules/rules_jvm_external/6.1/presubmit.yml

```diff
@@ -0,0 +1,37 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
```

Comment:
This would require replacing the `use_repo_rule` calls.

Reply:
That's what I thought. Will remove 6.x from this.